test/ostream-test.cc

```diff
@@ -191,3 +191,26 @@ TEST(OStreamTest, WriteToOStreamMaxSize) {
   } while (size != 0);
   fmt::internal::write(os, w);
 }
+
+#if __cplusplus >= 201103L
```

Comment:
This is needed because operator<< defined in ostream.h is only defined in this case.